slurp_XML_converted_JSON_to_solr.xslt

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- JSON converted into XML -->
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:java="http://xml.apache.org/xalan/java"
+  xmlns:xalan="http://xml.apache.org/xalan"
+  xmlns:foxml="info:fedora/fedora-system:def/foxml#">
+  <xsl:template match="foxml:datastream/foxml:datastreamVersion[last()][@MIMETYPE='application/json']" name="index_converted_json">
+    <xsl:param name="content"/>
+    <xsl:param name="prefix">
+      <xsl:value-of select="concat(../@ID, '_')"/>
+    </xsl:param>
+    <xsl:param name="suffix">ms</xsl:param>
+
+    <xsl:apply-templates mode="index_converted_json" select="$content/*[local-name() = 'json']">
```

Comment:
I'm thinking it could be and also should likely be configurable since it also is in `convertJSONToXML()`